oh_staff_ui/management/commands/create_mods_records.py

```diff
@@ -0,0 +1,35 @@
+import logging
+from django.core.management.base import BaseCommand
+from oh_staff_ui.classes.OralHistoryMods import OralHistoryMods
+from oh_staff_ui.models import ProjectItem
+
+# For handling command-line processing
+from django.contrib.auth.models import User
+
+
+logger = logging.getLogger(__name__)
```

Comment:
`logger` is not used.

Reply:
Noted, i'll remove for now, will be using in subsequent PR.